src/NServiceBus.Metrics.PerformanceCounters.Tests/IntegrationTests.cs

```diff
@@ -42,11 +42,13 @@ await endpoint.Stop()
             .ConfigureAwait(false);
 
         var criticalTimePerfCounter = new PerformanceCounter("NServiceBus", PerformanceCountersFeature.CriticalTimeCounterName, endpointName, true);
+        //var processingTimePerfCounter = new PerformanceCounter("NServiceBus", PerformanceCountersFeature.ProcessingTimeCounterName, endpointName, true);
```

Comment:
As it turned out these tests were pretty meaningless because they didn't detect the drop. Should we have a better test which tracks the value over time and verifies that the value drops? (with the drawback of longer test execution time)